From 5fe6b866594c6d59d1960356590a00ccc4cdf4c7 Mon Sep 17 00:00:00 2001
From: Zev Weiss <zev@bewilderbeest.net>
Date: Mon, 14 Oct 2019 23:38:02 -0500
Subject: [PATCH] alsactl: avoid needless wakeups in monitor loop.

The timeout wasn't really being used for anything; disabling it should
reduce idle energy consumption slightly.

Signed-off-by: Zev Weiss <zev@bewilderbeest.net>
Reviewed-by: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Tested-by: Takashi Sakamoto <o-takashi@sakamocchi.jp>
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 alsactl/monitor.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/alsactl/monitor.c b/alsactl/monitor.c
index cf4167bee165..fa6cd85d2ab2 100644
--- a/alsactl/monitor.c
+++ b/alsactl/monitor.c
@@ -340,7 +340,7 @@ static int run_dispatcher(int epfd, int sigfd, int infd, struct list_head *srcs,
 		int count;
 		int i;
 
-		count = epoll_wait(epfd, epev, max_ev_count, 200);
+		count = epoll_wait(epfd, epev, max_ev_count, -1);
 		if (count < 0) {
 			if (errno == EINTR)
 				continue;
-- 
2.16.4

